
 backend-2 Pulling 
 backend-1 Pulling 
 redis Pulling 
 9824c27679d3 Already exists 
 1a283093615e Already exists 
 c0055458ac5d Already exists 
 124e58f198c6 Pulling fs layer 
 23f500b479f6 Pulling fs layer 
 9824c27679d3 Already exists 
 1a283093615e Already exists 
 c0055458ac5d Already exists 
 124e58f198c6 Pulling fs layer 
 23f500b479f6 Pulling fs layer 
 124e58f198c6 Downloading [==================================================>]      93B/93B
 124e58f198c6 Verifying Checksum 
 124e58f198c6 Download complete 
 124e58f198c6 Extracting [==================================================>]      93B/93B
 124e58f198c6 Verifying Checksum 
 124e58f198c6 Download complete 
 124e58f198c6 Extracting [==================================================>]      93B/93B
 124e58f198c6 Extracting [==================================================>]      93B/93B
 124e58f198c6 Extracting [==================================================>]      93B/93B
 124e58f198c6 Pull complete 
 124e58f198c6 Pull complete 
 redis Pulled 
 23f500b479f6 Downloading [>                                                  ]  278.5kB/26.83MB
 23f500b479f6 Downloading [>                                                  ]  278.5kB/26.83MB
 23f500b479f6 Downloading [========>                                          ]  4.456MB/26.83MB
 23f500b479f6 Downloading [========>                                          ]  4.456MB/26.83MB
 23f500b479f6 Downloading [================>                                  ]  8.618MB/26.83MB
 23f500b479f6 Downloading [================>                                  ]  8.618MB/26.83MB
 23f500b479f6 Downloading [=======================>                           ]  12.78MB/26.83MB
 23f500b479f6 Downloading [=======================>                           ]  12.78MB/26.83MB
 23f500b479f6 Downloading [================================>                  ]  17.21MB/26.83MB
 23f500b479f6 Downloading [================================>                  ]  17.21MB/26.83MB
 23f500b479f6 Downloading [=======================================>           ]  21.37MB/26.83MB
 23f500b479f6 Downloading [=======================================>           ]  21.37MB/26.83MB
 23f500b479f6 Downloading [===============================================>   ]  25.52MB/26.83MB
 23f500b479f6 Downloading [===============================================>   ]  25.52MB/26.83MB
 23f500b479f6 Verifying Checksum 
 23f500b479f6 Download complete 
 23f500b479f6 Download complete 
 23f500b479f6 Extracting [>                                                  ]  294.9kB/26.83MB
 23f500b479f6 Extracting [>                                                  ]  294.9kB/26.83MB
 23f500b479f6 Extracting [=================>                                 ]  9.437MB/26.83MB
 23f500b479f6 Extracting [=================>                                 ]  9.437MB/26.83MB
 23f500b479f6 Extracting [==========================================>        ]     23MB/26.83MB
 23f500b479f6 Extracting [==========================================>        ]     23MB/26.83MB
 23f500b479f6 Extracting [=================================================> ]  26.54MB/26.83MB
 23f500b479f6 Extracting [=================================================> ]  26.54MB/26.83MB
 23f500b479f6 Extracting [==================================================>]  26.83MB/26.83MB
 23f500b479f6 Extracting [==================================================>]  26.83MB/26.83MB
 23f500b479f6 Extracting [==================================================>]  26.83MB/26.83MB
 23f500b479f6 Extracting [==================================================>]  26.83MB/26.83MB
 23f500b479f6 Pull complete 
 23f500b479f6 Pull complete 
 backend-1 Pulled 
 backend-2 Pulled 
 Network micheloliveira-com-dotnet-csharp-postgres_backend  Creating
 Network micheloliveira-com-dotnet-csharp-postgres_backend  Created
 Container micheloliveira-com-dotnet-csharp-postgres-redis-1  Creating
 Container micheloliveira-com-dotnet-csharp-postgres-backend-db-1  Creating
 Container micheloliveira-com-dotnet-csharp-postgres-backend-1-1  Creating
 Container micheloliveira-com-dotnet-csharp-postgres-backend-2-1  Creating
 Container micheloliveira-com-dotnet-csharp-postgres-backend-2-1  Created
 Container micheloliveira-com-dotnet-csharp-postgres-redis-1  Created
 Container micheloliveira-com-dotnet-csharp-postgres-backend-1-1  Created
 Container micheloliveira-com-dotnet-csharp-postgres-load-balancer-1  Creating
 Container micheloliveira-com-dotnet-csharp-postgres-backend-db-1  Created
 Container micheloliveira-com-dotnet-csharp-postgres-load-balancer-1  Created
Attaching to backend-1-1, backend-2-1, backend-db-1, load-balancer-1, redis-1
backend-1-1  | WORKER_SIZE: 10
backend-1-1  | BATCH_SIZE: 100
backend-1-1  | [Redis] Attempting connection...
backend-1-1  | Retry 1: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
backend-2-1  | WORKER_SIZE: 10
backend-2-1  | BATCH_SIZE: 100
backend-2-1  | [Redis] Attempting connection...
backend-2-1  | Retry 1: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
redis-1      | Starting Redis Server
backend-db-1  | The files belonging to this database system will be owned by user "postgres".
backend-db-1  | This user must also own the server process.
backend-db-1  | 
backend-db-1  | The database cluster will be initialized with locale "en_US.utf8".
backend-db-1  | The default database encoding has accordingly been set to "UTF8".
backend-db-1  | The default text search configuration will be set to "english".
backend-db-1  | 
backend-db-1  | Data page checksums are disabled.
backend-db-1  | 
backend-db-1  | fixing permissions on existing directory /var/lib/postgresql/data ... ok
backend-db-1  | creating subdirectories ... ok
backend-db-1  | selecting dynamic shared memory implementation ... posix
backend-1-1   | [Redis] Attempting connection...
backend-1-1   | Retry 2: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
backend-2-1   | [Redis] Attempting connection...
backend-2-1   | Retry 2: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
redis-1       | 1:C 21 Aug 2025 03:02:35.097 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
redis-1       | 1:C 21 Aug 2025 03:02:35.097 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis-1       | 1:C 21 Aug 2025 03:02:35.097 * Redis version=8.2.1, bits=64, commit=00000000, modified=1, pid=1, just started
redis-1       | 1:C 21 Aug 2025 03:02:35.097 * Configuration loaded
redis-1       | 1:M 21 Aug 2025 03:02:35.098 * monotonic clock: POSIX clock_gettime
redis-1       | 1:M 21 Aug 2025 03:02:35.098 * Running mode=standalone, port=6379.
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> RedisBloom version 8.2.0 (Git=unknown)
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> Registering configuration options: [
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> 	{ bf-error-rate       :      0.01 }
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> 	{ bf-initial-size     :       100 }
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> 	{ bf-expansion-factor :         2 }
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> 	{ cf-bucket-size      :         2 }
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> 	{ cf-initial-size     :      1024 }
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> 	{ cf-max-iterations   :        20 }
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> 	{ cf-expansion-factor :         1 }
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> 	{ cf-max-expansions   :        32 }
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * <bf> ]
redis-1       | 1:M 21 Aug 2025 03:02:35.099 * Module 'bf' loaded from /usr/local/lib/redis/modules//redisbloom.so
redis-1       | 1:M 21 Aug 2025 03:02:35.102 * <search> Redis version found by RedisSearch : 8.2.1 - oss
redis-1       | 1:M 21 Aug 2025 03:02:35.102 * <search> RediSearch version 8.2.1 (Git=dba8dd0)
redis-1       | 1:M 21 Aug 2025 03:02:35.102 * <search> Low level api version 1 initialized successfully
redis-1       | 1:M 21 Aug 2025 03:02:35.102 * <search> gc: ON, prefix min length: 2, min word length to stem: 4, prefix max expansions: 200, query timeout (ms): 500, timeout policy: return, cursor read size: 1000, cursor max idle (ms): 300000, max doctable size: 1000000, max number of search results:  1000000, 
redis-1       | 1:M 21 Aug 2025 03:02:35.102 * <search> Initialized thread pools!
redis-1       | 1:M 21 Aug 2025 03:02:35.102 * <search> Disabled workers threadpool of size 0
redis-1       | 1:M 21 Aug 2025 03:02:35.102 * <search> Subscribe to config changes
redis-1       | 1:M 21 Aug 2025 03:02:35.102 * <search> Enabled role change notification
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <search> Cluster configuration: AUTO partitions, type: 0, coordinator timeout: 0ms
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <search> Register write commands
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * Module 'search' loaded from /usr/local/lib/redis/modules//redisearch.so
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> RedisTimeSeries version 80200, git_sha=1439d4a439ca9c063e6ef124a510abff09a5d493
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> Redis version found by RedisTimeSeries : 8.2.1 - oss
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> Registering configuration options: [
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> 	{ ts-compaction-policy   :              }
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> 	{ ts-num-threads         :            3 }
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> 	{ ts-retention-policy    :            0 }
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> 	{ ts-duplicate-policy    :        block }
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> 	{ ts-chunk-size-bytes    :         4096 }
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> 	{ ts-encoding            :   compressed }
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> 	{ ts-ignore-max-time-diff:            0 }
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> 	{ ts-ignore-max-val-diff :     0.000000 }
redis-1       | 1:M 21 Aug 2025 03:02:35.103 * <timeseries> ]
redis-1       | 1:M 21 Aug 2025 03:02:35.104 * <timeseries> Detected redis oss
redis-1       | 1:M 21 Aug 2025 03:02:35.104 * <timeseries> Enabled diskless replication
redis-1       | 1:M 21 Aug 2025 03:02:35.104 * Module 'timeseries' loaded from /usr/local/lib/redis/modules//redistimeseries.so
redis-1       | 1:M 21 Aug 2025 03:02:35.104 * <ReJSON> Created new data type 'ReJSON-RL'
redis-1       | 1:M 21 Aug 2025 03:02:35.105 * <ReJSON> version: 80200 git sha: unknown branch: unknown
backend-db-1  | selecting default "max_connections" ... 100
backend-1-1   | [Redis] Attempting connection...
backend-1-1   | Retry 3: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
backend-2-1   | [Redis] Attempting connection...
backend-2-1   | Retry 3: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * <ReJSON> Exported RedisJSON_V1 API
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * <ReJSON> Exported RedisJSON_V2 API
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * <ReJSON> Exported RedisJSON_V3 API
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * <ReJSON> Exported RedisJSON_V4 API
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * <ReJSON> Exported RedisJSON_V5 API
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * <ReJSON> Enabled diskless replication
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * <ReJSON> Initialized shared string cache, thread safe: false.
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * Module 'ReJSON' loaded from /usr/local/lib/redis/modules//rejson.so
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * <search> Acquired RedisJSON_V5 API
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * Server initialized
redis-1       | 1:M 21 Aug 2025 03:02:35.195 * Ready to accept connections tcp
backend-1-1   | [Redis] Attempting connection...
backend-1-1   | [Redis] Connected successfully.
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 1: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Redis] Attempting connection...
backend-2-1   | [Redis] Connected successfully.
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 1: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | selecting default "shared_buffers" ... 128MB
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 2: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 2: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 3: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 3: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | selecting default time zone ... UTC
backend-db-1  | creating configuration files ... ok
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 4: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 4: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 5: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 5: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 6: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 6: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 7: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 7: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 8: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 8: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 9: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 9: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 10: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 10: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 11: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 11: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | running bootstrap script ... ok
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 12: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 12: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 13: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 13: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 14: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 14: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 15: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 15: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 16: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 16: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 17: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 17: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 18: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 18: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 19: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 19: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 20: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 20: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 21: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 21: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 22: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 22: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | sh: locale: not found
backend-db-1  | 2025-08-21 03:02:37.403 UTC [35] WARNING:  no usable system locales were found
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 23: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 23: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 24: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 24: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 25: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 25: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 26: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 26: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 27: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 27: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 28: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 28: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 29: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 29: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 30: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 30: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 31: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 31: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 32: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 32: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 33: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 33: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 34: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 34: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 35: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 35: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 36: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 36: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 37: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 37: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 38: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 38: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 39: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 39: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 40: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 40: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 41: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 41: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 42: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 42: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 43: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 43: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 44: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 44: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 45: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 45: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | performing post-bootstrap initialization ... ok
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 46: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 46: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | syncing data to disk ... ok
backend-db-1  | 
backend-db-1  | 
backend-db-1  | Success. You can now start the database server using:
backend-db-1  | 
backend-db-1  |     pg_ctl -D /var/lib/postgresql/data -l logfile start
backend-db-1  | 
backend-db-1  | initdb: warning: enabling "trust" authentication for local connections
backend-db-1  | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 47: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 47: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 48: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 48: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 49: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 49: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | waiting for server to start....2025-08-21 03:02:40.102 UTC [41] LOG:  starting PostgreSQL 17.6 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
backend-db-1  | 2025-08-21 03:02:40.102 UTC [41] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
backend-db-1  | 2025-08-21 03:02:40.104 UTC [44] LOG:  database system was shut down at 2025-08-21 03:02:39 UTC
backend-db-1  | 2025-08-21 03:02:40.108 UTC [41] LOG:  database system is ready to accept connections
backend-db-1  |  done
backend-db-1  | server started
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 50: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 50: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 51: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 51: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 52: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 52: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 53: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 53: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | CREATE DATABASE
backend-db-1  | 
backend-db-1  | 
backend-db-1  | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 54: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 54: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | CREATE TABLE
backend-db-1  | CREATE INDEX
backend-db-1  | 
backend-db-1  | 
backend-db-1  | waiting for server to shut down....2025-08-21 03:02:40.610 UTC [41] LOG:  received fast shutdown request
backend-db-1  | 2025-08-21 03:02:40.610 UTC [41] LOG:  aborting any active transactions
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 55: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 55: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | 2025-08-21 03:02:40.696 UTC [41] LOG:  background worker "logical replication launcher" (PID 47) exited with exit code 1
backend-db-1  | 2025-08-21 03:02:40.696 UTC [42] LOG:  shutting down
backend-db-1  | 2025-08-21 03:02:40.697 UTC [42] LOG:  checkpoint starting: shutdown immediate
backend-db-1  | 2025-08-21 03:02:40.706 UTC [42] LOG:  checkpoint complete: wrote 927 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.009 s, sync=0.001 s, total=0.010 s; sync files=0, longest=0.000 s, average=0.000 s; distance=4276 kB, estimate=4276 kB; lsn=0/191A368, redo lsn=0/191A368
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 56: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 56: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | 2025-08-21 03:02:40.800 UTC [41] LOG:  database system is shut down
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 57: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 57: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  |  done
backend-db-1  | server stopped
backend-db-1  | 
backend-db-1  | PostgreSQL init process complete; ready for start up.
backend-db-1  | 
backend-1-1   | [Postgres] Attempting warmup connection...
backend-1-1   | Async Retry 58: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-2-1   | [Postgres] Attempting warmup connection...
backend-2-1   | Async Retry 58: NpgsqlException - Failed to connect to 172.21.0.4:5432
backend-db-1  | 2025-08-21 03:02:41.003 UTC [1] LOG:  starting PostgreSQL 17.6 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
backend-db-1  | 2025-08-21 03:02:41.003 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
backend-db-1  | 2025-08-21 03:02:41.003 UTC [1] LOG:  listening on IPv6 address "::", port 5432
backend-db-1  | 2025-08-21 03:02:41.003 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
backend-db-1  | 2025-08-21 03:02:41.005 UTC [59] LOG:  database system was shut down at 2025-08-21 03:02:40 UTC
backend-db-1  | 2025-08-21 03:02:41.009 UTC [1] LOG:  database system is ready to accept connections
backend-1-1   | [Postgres] Attempting warmup connection...
backend-2-1   | [Postgres] Attempting warmup connection...
backend-1-1   | [Postgres] Connection warmup successful.
backend-2-1   | [Postgres] Connection warmup successful.
load-balancer-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
load-balancer-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
load-balancer-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
load-balancer-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
load-balancer-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
load-balancer-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
load-balancer-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
load-balancer-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
load-balancer-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 16) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1428, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-db-1     | 2025-08-21 03:07:41.102 UTC [57] LOG:  checkpoint starting: time
[Kredis-1 exited with code 137
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1544, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 8, async-ops: 145604, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=8,QueuedItems=1,CompletedItems=440091,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1542, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 8, async-ops: 145604, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=4,Free=32763,Min=1,Max=32767), POOL: (Threads=8,QueuedItems=1,CompletedItems=440091,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1545, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 8, async-ops: 145604, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=1,Free=32766,Min=1,Max=32767), POOL: (Threads=8,QueuedItems=0,CompletedItems=440091,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1534, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 8, async-ops: 145604, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=5,Free=32762,Min=1,Max=32767), POOL: (Threads=8,QueuedItems=2,CompletedItems=440091,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1543, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 8, async-ops: 145604, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=8,QueuedItems=1,CompletedItems=440091,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1533, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 8, async-ops: 145604, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=5,Free=32762,Min=1,Max=32767), POOL: (Threads=8,QueuedItems=3,CompletedItems=440091,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1540, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 8, async-ops: 145604, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=5,Free=32762,Min=1,Max=32767), POOL: (Threads=8,QueuedItems=1,CompletedItems=440091,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at CountingHandler.<SendAsync>d__8.MoveNext() + 0xa0
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<<SendCoreAsync>g__Core|4_0>d.MoveNext() + 0x13c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at System.Net.Http.HttpClient.<<SendAsync>g__Core|83_0>d.MoveNext() + 0x399
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentProcessorService.<ProcessPaymentAsync>d__26.MoveNext() + 0x27e
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at PaymentService.<ProcessPaymentAsync>d__22.MoveNext() + 0x15a
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x43d
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1502, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 8, async-ops: 145604, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=5,Free=32762,Min=1,Max=32767), POOL: (Threads=8,QueuedItems=5,CompletedItems=440091,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 8) on redis:6379/Interactive, Idle/Faulted, last: RPUSH, origin: ReadFromPipe, outstanding: 1426, last-read: 4s ago, last-write: 0s ago, unanswered-write: 4s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x544
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Distributed.Redis.ReactiveLockRedisTrackerStore.<SetStatusAsync>d__10.MoveNext() + 0x9c
backend-1-1      | --- End of stack trace from previous location ---
backend-1-1      |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__13.MoveNext() + 0x68
log truncated at line 1000
